src/style.css

```diff
@@ -1,3 +1,161 @@
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+img {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+button {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+:root {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --white-accent: #fff;
+  --secondary: #616070;
+}
+
 body {
+  margin: auto;
+}
+
+.card {
+  margin: auto;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  background-color: var(--white-accent);
+}
+
+.card__goods {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 0;
+  display: block;
+
+}
+
+.card__info {
+  color: var(--main-accent);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 16px 0;
+  width: 166px;
+
+}
+
+.card__code {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--secondary);
+
+  padding: 4px 84px 16px 16px;
   margin: 0;
 }
+
+.card-chosen {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+  align-items: flex-end;
+  height: 16px;
+}
+
+.card-reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--main-accent);
+}
+
+.card__buy {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 16px;
+  align-items: center;
+  height: 18px;
+}
+
+.card__price {
+  color: var(--secondary);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__sum {
+  /* font-family: Roboto, sans-serif; */
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.card__button {
+  color: var(--white-accent);
+  height: 40px;
+  width: 166px;
+  border-radius: 5px;
+  background-color: var(--blue-accent);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  margin-bottom: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
```

Comment:
You don't need to write the left and right margin for each element separately, instead, you can just write the inner margin for the card
```suggestion
```